src/main/java/funda/prog101/Pot.java

```diff
@@ -0,0 +1,5 @@
+package main.java.funda.prog101;
+
+public class Pot {
```

Comment:
In the diagram Class it was defined like a extends of Container, it would be public class Pot extends Contatiner